`watch` command fails when trying to join aliases

For example, doing a `!mjolnir watch #matrix-org-hs-tos-bl:matrix.org` causes the bot to resolve the alias, and then try to join the room without any via parameters (that I can see):

```
[INFO] [Mjolnir] Command being run by @foobar:example.com: !mjolnir watch #matrix-org-coc-bl:matrix.org
[ERROR] [CommandHandler] { errcode: 'M_UNKNOWN', error: 'No known servers' }
[ERROR] [MatrixHttpClient] (REQ-38579) [Error: Error during MatrixClient request POST /_matrix/client/v3/join/!WuBtumawCeOGEieRrp%3Amatrix.org: 404 Not Found -- {"errcode":"M_UNKNOWN","error":"No known servers"}]
```

--- a/src/models/PolicyList.ts
+++ b/src/models/PolicyList.ts
@@ -663,8 +663,7 @@ export class PolicyListManager {
         if (permalink.roomIdOrAlias.startsWith("!")) {
             roomId = permalink.roomIdOrAlias
             viaServers = permalink.viaServers
-        }
-        else {
+        } else {
             const roomInfo = await this.mjolnir.client.lookupRoomAlias(permalink.roomIdOrAlias)
             roomId = roomInfo.roomId
             viaServers = roomInfo.residentServers

--- a/src/models/PolicyList.ts
+++ b/src/models/PolicyList.ts
@@ -658,9 +658,20 @@ export class PolicyListManager {
         const permalink = Permalinks.parseUrl(roomRef);
         if (!permalink.roomIdOrAlias) return null;
 
-        const roomId = await this.mjolnir.client.resolveRoom(permalink.roomIdOrAlias);
+        let roomId: string;
+        let viaServers;
+        if (permalink.roomIdOrAlias.startsWith("!")) {
+            roomId = permalink.roomIdOrAlias
+            viaServers = permalink.viaServers
+        }
+        else {
+            const roomInfo = await this.mjolnir.client.lookupRoomAlias(permalink.roomIdOrAlias)
+            roomId = roomInfo.roomId
+            viaServers = roomInfo.residentServers
+        }
+
         if (!joinedRooms.includes(roomId)) {
-            await this.mjolnir.client.joinRoom(roomId, permalink.viaServers);
+            await this.mjolnir.client.joinRoom(roomId, viaServers);
         }
 
         if (this.policyLists.find(b => b.roomId === roomId)) {

